### PR TITLE
initial application structure

### PR DESCRIPTION
**Pull request by jonnyalexbh**
## Summary
- initial application structure

## Trello Card
N/A

## Screenshot
Screen Shot 2019-12-09 at 11 11 55 PM

### CODE REVIEW

**Review by betoobernal — COMMENTED** [review]
- Please, remove package-lock.json
- Use wolox/equalizer in devDependencies

File: plink-frontend/src/app/app.component.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  margin: 0 auto;
+  max-width: 85%;
+  position: relative;
+  width: 100%;
+  &::after {
+    background: url(../../src/assets/footer.png) no-repeat;
+    background-position: bottom;
+    background-repeat: no-repeat;
+    background-size: 100%;
+    bottom: 28%;
```
Comment (betoobernal) [inline plink-frontend/src/app/app.component.scss:18]: missing horizontal position property (right | left)

File: plink-frontend/src/app/app.component.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    bottom: 0;
+    content: '';
+    height: 20px;
+    left: 0;
+    position: absolute;
+    width: 100%;
+  }
+}
+
+.content-active {
```
Comment (betoobernal) [inline plink-frontend/src/app/app.component.scss]: Use *ngIf instead.

File: plink-frontend/src/app/app.component.scss (hunk 1 above)
Comment (jonnyalexbh) [reply plink-frontend/src/app/app.component.scss:18]: 👍

File: plink-frontend/src/app/app.component.scss (hunk 2 above)
Comment (jonnyalexbh) [reply plink-frontend/src/app/app.component.scss]: ready 👍

**Review by betoobernal — APPROVED** [review]

**Pull request merged**